recipe_scrapers/_abstract.py

```diff
@@ -15,30 +17,25 @@
 }
 
 
-class AbstractScraper(metaclass=PluginsRunner):
+class AbstractScraper:
     def __init__(
         self,
         url,
-        exception_handling: bool = True,
-        meta_http_equiv: bool = False,
         proxies: Optional[str] = None,  # allows us to specify optional proxy server
-        test: bool = False,
-        timeout: Union[
-            float, Tuple, None
+        timeout: Optional[
```

Comment:
I think `Optional` around `Union` that has `None` is redundant?

Reply:
haha probably it is 😅  I'll leave the typing to someone else for sure